### DISCUSSION TITLE
Move quartz extension to stable?

### DISCUSSION THREAD

**Discussion by alex-martel** [quarkusio/quarkus#44778] [Community]
Somewhere in 2022 the quartz extension was marked stable, but this was reverted here https://github.com/quarkusio/quarkus/pull/27453 due to quartz using javax packages + not being actively maintained. Since this recently changed, will the extension be marked as stable?

**Comment by mkouba** [comment]
FYI there was a long discussion on zulip: https://quarkusio.zulipchat.com/#narrow/channel/187038-dev/topic/.E2.9C.94.20Quartz.3A.20Forking.20an.20open.20source.20repo.20dilemma

You're right that `quarkus-quartz` was marked as stable in 2022, but then reverted to preview in https://github.com/quarkusio/quarkus/pull/27453. Since then a few things changed, a new Quartz version was released and Quarkus was updated in https://github.com/quarkusio/quarkus/pull/44477. So I think that it's on the right track.

WDYT @gsmet @melloware @maxandersen

**Reply by melloware** [reply]
Yes please . Stable now that 3.17 is using the latest Quartz which is Jakarta-fied and everything seems to be working great!
